### ISSUE TITLE
Is there a way to set the width of the SDL window?

### ISSUE THREAD

**Issue by Kirtai** [dseagrav/ld#39]
As the title asks, is there a way to change the SDL window width?

I've read that the original monitor was 800x1024 (portrait) and would like to set it to that. (I tile my windows to half the screen and it's slightly oversized for a standard HD display.)

**Comment by Devon7** [comment]
`(tv:set-console-size 800 1024)` **;** is this what you have in mind?

**Comment by Kirtai** [comment]
No, that changes the area used by the emulator, but the overall window still opens at 1024 pixels wide. I was hoping for a way to tell the initial SDL window to open at 800 pixels wide, without having to manually change it after opening.

**Comment by Devon7** [comment]
We'll configure the emulator to know where the microcode screen parameters are,
just as we configure it to know where the mouse coordinates are,
and automagically resize.

**Comment by Kirtai** [comment]
Thank you, that sounds ideal :)

**Comment by bictorv** [comment]
(The Lambda monitor was landscape, while the CADR monitor was portrait.)
